code/modules/mob/living/basic/space_fauna/netherworld/blankbody.dm

```diff
@@ -20,7 +20,10 @@
 	unsuitable_atmos_damage = 0
 	unsuitable_cold_damage = 0
 	unsuitable_heat_damage = 0
-	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+	// Real blue, trying to go for the migo's look
```

Comment:
This isn't the migo, it's the blankbody. If the numbers are fine it's fine, but just alerting you to that in case some conflict spurred this mis-comment.

Reply:
yeah I know they split migo's into another subtype. I may just add a new comment for these since I am still happy with the coloring

Reply:
added new colors for these